### PR TITLE
[zh] sync concepts\windows\intro.md

### PR DESCRIPTION
**Pull request by ydFu**
The file `content/en/docs/concepts/windows/intro.md` has been updated in this PR (https://github.com/kubernetes/website/pull/45612) ,so we will synchronize the localization.

### CODE REVIEW

File: content/zh-cn/docs/concepts/windows/intro.md [hunk 1]
```diff
@@ -772,7 +772,28 @@ troubleshooting ideas prior to creating a ticket.
...
+It can be used to validate all the functionalities of a Windows and hybrid system
+(mixed with Linux nodes) with full coverage.
+
+To set up the project on a newly created cluster, refer to the instructions in the
+[project guide](https://github.com/kubernetes-sigs/windows-operational-readiness/blob/main/README.md).
+-->
+
+### 验证 Windows 集群的操作性  {#validating-windows-cluster-operability}
+
+Kubernetes 项目提供了 *Windows 操作准备* 规范，配备了结构化的测试套件。
```
Comment (windsonsea) [inline content/zh-cn/docs/concepts/windows/intro.md:790]:
```suggestion
Kubernetes 项目提供了 **Windows 操作准备** 规范，配备了结构化的测试套件。
```

File: content/zh-cn/docs/concepts/windows/intro.md [hunk 2]
```diff
@@ -772,7 +772,28 @@ troubleshooting ideas prior to creating a ticket.
...
+这个套件分为两组测试，核心和扩展，每组都包含针对特定领域测试的分类。
```
Comment (windsonsea) [inline content/zh-cn/docs/concepts/windows/intro.md:791]:
```suggestion
这个套件分为两组测试：核心和扩展。每组测试都包含了针对特定场景的分类测试。
```

File: content/zh-cn/docs/concepts/windows/intro.md [hunk 3]
```diff
@@ -772,7 +772,28 @@ troubleshooting ideas prior to creating a ticket.
...
+它可以用来验证 Windows 和混合系统（混合了 Linux 节点）的所有功能，实现全面覆盖。
+
+要在新创建的集群上设置项目，请参考
+[项目指南](https://github.com/kubernetes-sigs/windows-operational-readiness/blob/main/README.md)中的说明。
```
Comment (windsonsea) [inline content/zh-cn/docs/concepts/windows/intro.md:795]:
```suggestion
要在新创建的集群上搭建此项目，
请参考[项目指南](https://github.com/kubernetes-sigs/windows-operational-readiness/blob/main/README.md)中的说明。
```

**Review by howieyuen — COMMENTED** [review]
/lgtm

**Pull request merged**